_scripts/instruction-widget/templates/getting-started/renewal.html

```diff
@@ -18,16 +18,11 @@ <h3>Automating renewal</h3>
 </pre>
 
 If that appears to be working correctly, you can arrange for automatic renewal
-by adding a <tt>cron</tt> or <tt>systemd</tt> job which runs the
+by adding a <tt><a href="http://www.unixgeeks.org/security/newbie/unix/cron-1.html">cron</a></tt> or <tt><a href="https://en.wikipedia.org/wiki/Systemd">systemd</a></tt> job which runs the
 following:<br>
 
 <pre>{{base_command}} renew {{#cbauto}}--no-self-upgrade{{/cbauto}}</pre>
 
-<p>More detailed information and options about renewal can be found in <a
-href="https://certbot.eff.org/docs/using.html#renewal">the full
-documentation</a>.</p>
-
-{{^cron_included}}
```

Comment:
Rather than deleting these lines, I think we should actually expand their scope a bit. If cron is already configured on your system, we don't need to include text about how to set it up. I'd put this after the `--dry-run` command and put the closing tag after the example cron job.